### ISSUE TITLE
Incompatible with Redmine4.1.0

### ISSUE THREAD

**Issue by gainskills** [a-ono/redmine_ckeditor#312]
Redmine4.1.0 released on 22-Dec-2019

The redmine is unable to bootup with the plugin: Redmine-ckeditor.

Expectation:
Redmine-ckeditor is compatible for Redmine4.1.0.

Error log when Redmine is booting up:
```
redmine_1     | /usr/src/redmine/plugins/redmine_ckeditor/lib/tasks/ckeditor.rake:23: warning: previous definition of FORMATS was here
redmine_1     | rake aborted!
redmine_1     | NoMethodError: undefined method `assets' for #<Rails::Application::Configuration:0x0000561fa9ffe730>
redmine_1     | Did you mean?  asset_host
redmine_1     | /usr/local/bundle/gems/railties-5.2.4.1/lib/rails/railtie/configuration.rb:97:in `method_missing'
redmine_1     | /usr/local/bundle/bundler/gems/rich-15f6353d2c3d/lib/rich/engine.rb:9:in `block in <class:Engine>'
redmine_1     | /usr/local/bundle/gems/railties-5.2.4.1/lib/rails/initializable.rb:32:in `instance_exec'
redmine_1     | /usr/local/bundle/gems/railties-5.2.4.1/lib/rails/initializable.rb:32:in `run'
redmine_1     | /usr/local/bundle/gems/railties-5.2.4.1/lib/rails/initializable.rb:61:in `block in run_initializers'
redmine_1     | /usr/local/bundle/gems/railties-5.2.4.1/lib/rails/initializable.rb:60:in `run_initializers'
redmine_1     | /usr/local/bundle/gems/railties-5.2.4.1/lib/rails/application.rb:361:in `initialize!'
redmine_1     | /usr/src/redmine/config/environment.rb:16:in `<top (required)>'
redmine_1     | /usr/local/bundle/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:291:in `require'
redmine_1     | /usr/local/bundle/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:291:in `block in require'
redmine_1     | /usr/local/bundle/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:257:in `load_dependency'
redmine_1     | /usr/local/bundle/gems/activesupport-5.2.4.1/lib/active_support/dependencies.rb:291:in `require'
redmine_1     | /usr/local/bundle/gems/railties-5.2.4.1/lib/rails/application.rb:337:in `require_environment!'
redmine_1     | /usr/local/bundle/gems/railties-5.2.4.1/lib/rails/application.rb:520:in `block in run_tasks_blocks'
redmine_1     | /usr/local/bundle/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
redmine_1     | Tasks: TOP => db:migrate => db:load_config => environment
redmine_1     | (See full trace by running task with --trace)
````
Thanks,

**Comment by CNPF** [comment]
@gainskills did you find the solution?

**Comment by gainskills** [comment]
@CNPF unfortunately,  I don't have the solution with my limited knowledge on Ruby

**Comment by cjsaohan** [comment]
I hope this problem can be solved as soon as possible

**Comment by davey** [comment]
+1 - looking for a solution on this too...

**Comment by kaitaku** [comment]
+1

**Comment by tmuellerdatabay** [comment]
+1

**Comment by a-ono** [comment]
Fixed in version 1.2.3.
